modules/plugins/lsp/harper-ls/harper-ls.nix

```
@@ -0,0 +1,35 @@
+{lib, ...}: let
+  inherit (lib.options) mkOption mkEnableOption;
+  inherit (lib.types) anything attrsOf nullOr;
+in {
+  options.vim.lsp.harper-ls = {
+    enable = mkEnableOption "Harper grammer checking LSP";
+    settings = mkOption {
+      type = nullOr (attrsOf anything);
+      default = null;
```

is null necessary? passing empty table `{}` to `vim.lsp.servers.harper-ls.settings` should be fine
```suggestion
      type = attrsOf anything;
      default = {};
```

---

Done.